BooruSharp/Search/Post/SearchResult.cs

```diff
@@ -73,6 +82,8 @@ namespace BooruSharp.Search.Post
         /// Gets the read-only collection containing all the tags associated with the file.
         /// </summary>
         public ReadOnlyCollection<string> Tags { get; }
+        
+        public ReadOnlyCollection<Tag.SearchResult> DetailedTags { get; }
```

Comment:
Could you add the XML documentation to explain what this is?

Reply:
Absolutely, just stopping to ask if you even like me adding stuff.

Reply:
Ah yup I even forgot to answer that
Yeah, I do appreciate it, thanks for taking time contributing to this project :)

Reply:
I'll be working on this more then